R/rdeephaven/R/update_by_ops_wrapper.R

```diff
@@ -0,0 +1,615 @@
+#' Default is to compute the exponential moving standard deviation for all non-grouping columns.
+#' @param operation_control OperationControl that defines how special cases will behave. See `?OperationControl` for more information.
+#' @export
+udb_emstd_tick <- function(decay_ticks, cols = character(), operation_control = op_control()) {
+  verify_real("decay_ticks", decay_ticks, TRUE)
+  verify_string("cols", cols, FALSE)
+  verify_type("operation_control", operation_control, TRUE, "OperationControl", "a Deephaven OperationControl")
+  return(UpdateByOp$new(INTERNAL_emstd_tick(decay_ticks, cols, operation_control$.internal_rcpp_object)))
+}
+
+#' @description
+#' Creates an exponential moving standard deviation UpdateByOp for each column in `cols`, using time as the decay unit.
+#' The formula used is
+#'   `a = e^(-dt / decay_time)`
+#'   `ema_next = a * ema_last + (1 - a) * value`
+#'   `em_variance_next = a * (em_variance_last + (1 − a) * (value − ema_last)^2)`
+#'   `emstd_next = sqrt(em_variance_next)`
+#' @param ts_col String denoting the column to use as the timestamp.
+#' @param decay_time ISO-8601-formatted string specifying the decay rate.
+#' @param cols String or list of strings denoting the column(s) to operate on. Can be renaming expressions, i.e. “new_col = col”.
+#' Default is to compute the exponential moving standard deviation for all non-grouping columns.
+#' @param operation_control OperationControl that defines how special cases will behave. See `?OperationControl` for more information.
+#' @export
+udb_emstd_time <- function(ts_col, decay_time, cols = character(), operation_control = op_control()) {
+  verify_string("ts_col", ts_col, TRUE)
+  verify_string("decay_time", decay_time, TRUE)
+  verify_string("cols", cols, FALSE)
+  verify_type("operation_control", operation_control, TRUE, "OperationControl", "a Deephaven OperationControl")
+  return(UpdateByOp$new(INTERNAL_emstd_time(ts_col, decay_time, cols, operation_control$.internal_rcpp_object)))
+}
+
+#' @description
+#' Creates a rolling sum UpdateByOp for each column in `cols`, using ticks as the windowing unit.
+#' Ticks are row counts, and you may specify the reverse and forward window in number of rows to include.
+#' The current row is considered to belong to the reverse window but not the forward window.
+#' Also, negative values are allowed and can be used to generate completely forward or completely reverse windows.
+#' @param cols String or list of strings denoting the column(s) to operate on. Can be renaming expressions, i.e. “new_col = col”.
+#' Default is to compute the rolling sum for all non-grouping columns.
+#' @param rev_ticks Integer scalar denoting the look-behind window size in number of rows.
+#' @param fwd_ticks Integer scalar denoting the look-ahead window size in number of rows. Default is 0.
+#' @export
+udb_roll_sum_tick <- function(cols, rev_ticks, fwd_ticks = 0) {
```

Comment:
In all of these, `rolling` was shortened to `roll`.  I'm not sure what I think.  It will make it harder to find similar docs from other langs, so I might bias to `rolling`, even though it is slightly longer.